ksqldb-common/src/test/java/io/confluent/ksql/schema/ksql/LogicalSchemaTest.java

```diff
@@ -239,6 +257,26 @@ public void shouldPreferValueOverMetaColumns() {
     );
   }
 
+  @Test
+  public void shouldPreferValueOverRowPartitionAndOffsetColumns() {
```

Comment:
nit: let's just merge this into the existing `shouldPreferValueOverMetaColumns()` test rather than adding a new one. This test doesn't depend on pseudocolumn version so the two tests are doing the same thing.